src/openzaak/components/documenten/tests/test_reserved_document.py

```diff
@@ -0,0 +1,287 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2020 Dimpact
+from datetime import date
+
+from freezegun import freeze_time
+from rest_framework import serializers, status
+from rest_framework.test import APITestCase
+from vng_api_common.constants import ComponentTypes
+from vng_api_common.tests import reverse
+
+from openzaak.components.autorisaties.management.commands.sync_autorisaties import (
+    Autorisatie,
+)
+from openzaak.components.documenten.api.scopes import SCOPE_DOCUMENTEN_AANMAKEN
+from openzaak.components.documenten.api.serializers import ReservedDocumentSerializer
+from openzaak.components.documenten.api.utils import generate_document_identificatie
+from openzaak.components.documenten.models import ReservedDocument
+from openzaak.components.documenten.tests.factories import (
+    EnkelvoudigInformatieObjectFactory,
+)
+from openzaak.tests.utils import JWTAuthMixin
+
+
+@freeze_time("2025-01-01T12:00:00")
+class ReservedDocumentTests(JWTAuthMixin, APITestCase):
+    scopes = [SCOPE_DOCUMENTEN_AANMAKEN]
+
+    def setUp(self):
+        super().setUp()
+
+        self.bronorganisatie = "812345678"
+        self.reserved_identificatie = "DOCUMENT-2025-0000000001"
+        self.new_identificatie = "DOCUMENT-2025-0000000002"
+        self.another_reserved_identificatie = "DOCUMENT-2025-0000000003"
+        self.url = reverse("reserveddocument-list")
+
+        self.reserved_document = ReservedDocument.objects.create(
+            identificatie=self.reserved_identificatie,
+            bronorganisatie=self.bronorganisatie,
+        )
+
+        ReservedDocument.objects.create(
+            identificatie=self.another_reserved_identificatie,
+            bronorganisatie=self.bronorganisatie,
+        )
+
+        Autorisatie.objects.create(
+            applicatie=self.applicatie,
+            component=ComponentTypes.drc,
+            scopes=["documenten.aanmaken"],
+        )
```

Comment:
If we have a separate testcase for the permission part, we can use `heeft_alle_autorisaties=True` at the class level instead

```suggestion
```